src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java

```diff
@@ -55,6 +72,26 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         return result;
     }
 
+    /* This method gets the numbers of instructors that would be made visible to students if
+    changes to the instructor information were successfully processed. Thus, if we are
+    trying to make the instructor false, we need to account for that.
+     */
+    protected int getNumberOfInstructorsDisplayedToStudents(String courseId, boolean currentlyDisplayed,
```

Comment:
For comments for a method, use the javadoc style for the asterisks, ie:

```java
/**
 * A comment about the method
 * spanning multiple lines
 */
```

This allows most IDEs supporting Java to correctly classify it as a comment for the method, allowing it to be placed in areas where it would be helpful (eg. as a tooltip on hover for Eclipse).